enterprise/api/v1/views.py

```diff
@@ -302,7 +300,7 @@ class EnterpriseCustomerCatalogViewSet(EnterpriseReadOnlyModelViewSet):
 
     @permission_required('enterprise.can_view_catalog', fn=lambda request, *args, **kwargs: None)
     def list(self, request, *args, **kwargs):
-        return super(EnterpriseCustomerCatalogViewSet, self).list(request, *args, **kwargs)
+        return super(EnterpriseCustomerCatalogViewSet, self).list(request, *args, **kwargs)  # pylint: disable=no-member
```

Comment:
You can add the `pylint: disable=no-member` suppression on the top of file just once instead of adding it on each line separately.